src/main/webapp/repo-bootstrap/global/blueprints/1000_website_editorial/config/studio/site-config.xml

```diff
@@ -69,6 +69,16 @@
         </ignore-postfix-fields>
     </form-engine>
 
+    <!-- Site Translation -->
```

Comment:
Why adding configuration as commented?

Reply:
According to the ticket it should be added as commented for backwards compatibility

Reply:
It is a new feature. There is no backward compatibility. Old code does not know about that config so it will never read it if it exists. Only a new code needs to handle config if it does not exist. The config file should have it or not have it. Why adding a new config as a comment? There is a sample file where you added it as an example of config.

Reply:
I agree, updating the sample should be enough, what do you think @sumerjabri?

Reply:
I want the block added and commented out. This way existing sites can uncomment and use the new feature.